### DISCUSSION TITLE
Routing to backend urls which are generated at runtime

### DISCUSSION THREAD

**Discussion by archanak-msft** [dotnet/yarp#464] [General] [answered]
We have a requirement where we want to route the incoming requests to backend servers whose urls might be different for every incoming request. Just wondering if this reverse-proxy has a way to achieve this.

Thanks in advance for your help.

**Comment by Tratcher** [comment, marked as answer]
Check out https://github.com/microsoft/reverse-proxy/pull/455 which adds something for this in preview6.

**Reply by archanak-msft** [reply]
Thanks, this helps.
